### DISCUSSION TITLE
Lacking Notification for PRs assigned to Teams

### DISCUSSION THREAD

**Discussion by sfc-gh-dfrail** [community/community#123333] [Mobile]
Hello. 
For most PRs, my team/group is automatically assigned as an approver/reviewer, but GitHub mobile app fails to send notifications unless I'm directly tagged.
Would be great if this could be resolved.

**Comment by mattruggio** [comment]
Hey @sfc-gh-dfrail, thanks for the heads up. Can you please respond back with your phone/model and GitHub app version you are running? Also, are you referring specifically to push notifications regarding team assignment or are you referring to the team assignment notification not showing up in your notifications inbox, or both?

**Reply by sfc-gh-dfrail** [reply]
Hi @mattruggio. 
I'm currently using Samsung S22U and on GH app version 1.161.0, but my colleague who uses an iPhone noted the same issue.
I get push notification when someone directly tags me to review PRs:
<img width="297" alt="image" src="https://github.com/community/community/assets/106282821/a41188c5-c755-4adc-b2ea-50ac4f091d6c">
But when my team is tagged, the request shows up in `Notifications` tab but I don't get push notification from the app, hence it's easy to miss those PRs. Would this be replicable on your end?

**Reply by mattruggio** [reply]
Hey @sfc-gh-dfrail, I just looked into this and I believe this is correct functionality: team mentions will not create push notifications for mobile devices, but the team mentions will be available in your Notifications inbox. If I had to guess, I would assume this was an anti-spam measure done so to not create a flood of (potentially unwanted) push notifications. You are correct that once you are directly mentioned or assigned then it should send you a push notification.

**Reply by sfc-gh-dfrail** [reply]
@mattruggio Ok, thank you for confirming. Will have to use Slack integration to get notifications then. 
I would've expected GH app to allow same configurations like it does for Slack:
<img width="450" alt="image" src="https://github.com/community/community/assets/106282821/a8f6d31e-1042-4221-ba49-bec1cced7521">
